Chap_API_Sync_Access.tex

```diff
@@ -284,12 +285,9 @@ \subsection{\code{PMIx_Get_nb}}
 \argin{cbdata}{Data to be passed to the callback function (memory reference)}
 \end{arglist}
 
-Returns one of the following:
+\returnsimplenb
 
-\begin{itemize}
-    \item \refconst{PMIX_SUCCESS}, indicating that the request is being processed by the host environment - result will be returned in the provided \refarg{cbfunc}. Note that the library must not invoke the callback function prior to returning from the \ac{API}.
-    \item a \ac{PMIx} error constant indicating either an error in the input or that the request was immediately processed and failed - the \refarg{cbfunc} will \textit{not} be called.
-\end{itemize}
+The callback function \refarg{cbfunc} is only called when \refconst{PMIX_SUCCESS} is returned.
```

Comment:
Revision Exception:
```suggestion
```

Sentence is redundant with the `returnsimplenb` contents.